config/initializers/services.rb

```diff
@@ -36,3 +37,11 @@ def assign_db(lhs, rhs)
 end
 
 Services.register(:whois) { Whois::Client.new }
+
+geoip_path = if Rails.env.production?
+  "/htprep/geoip/GeoLite2-City.mmdb"
```

Comment:
We already have the per-environment configuration facility, and the ability to override it however we like, so we don't need to worry about this here:

```ruby
Services.register(:geoip)  { MaxMind::GeoIP2::Reader.new(Otis.config.geoip_path) }
```

and in `config/settings.yml` add:

```ruby
geoip_path: <%= Rails.root.join("vendor/geoip/GeoIP2-City-Test.mmdb" %>
```

Reply:
After some further investigation, looks like Ettin does not like ERB interpolations that reference `Rails`. I can pull that `geoip_path` logic out into `application.rb` which is more config-centric. Maybe stick the production `htprep` path into `settings.yml` as a default (or do we want to get that from `ENV` only?). But alas I can propose nothing of real elegance.

Reply:
I see -- probably an issue with load/initialization order.

I would say that since in dev we're running in a container, we can just default to the full path:

```yaml
geoip_path: /usr/src/app/vendor/geoip/GeoIP2-City-Test.mmdb
```

Reply:
After a randomly failing test (weird) looks like that works.